backend/src/server/context.ts

```diff
@@ -18,13 +19,16 @@ export interface Context {
 
 const context = (args: ExpressContext): Context => {
   const authorization = args.req.headers.authorization
-  let token: string | null = null
+  const clientRequestTime = args.req.headers.clientrequesttime
```

Comment:
clientrequesttime now all in lowercase is correct?

Reply:
Yes, it looks like the request headers are all lowercase. I tried it first in camel case